### PR TITLE
docs(v2): proofread docs for alpha.20

### PR DESCRIPTION
**Pull request by yangshun**
## Motivation

- Change headings from Title Case to Sentence case
- Proofread directory structure for new `/src/` directory
- Fix a few typos

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

See Netlify

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by yangshun** [comment]
@wgao19 this will conflict with https://github.com/facebook/docusaurus/pull/1645 (I renamed a file) but I guess it's not too big of a deal as that PR doesn't contain too many things yet.

File: website/docs/creating-pages.md [hunk 1]
```diff
@@ -39,16 +39,35 @@ export default Hello;
...
+1. Add a `/src/pages/support.js` file
+1. Create a `/src/pages/support/` directory and a `/src/pages/support/index.js` file. This has the benefits of letting you create a CSS module file (`styles.module.css`) for `support.js` that is near where it's being used. **Note:** you will have to manually import the CSS module file within your component module.
+
+```sh
+my-website
+├── src
+│   └── pages
+│       ├── styles.module.css
+│       └── index.js
+│           └── support
```
Comment (wgao19) [inline website/docs/creating-pages.md]: this should be parallel to index.js right?

File: website/docs/creating-pages.md [hunk 2]
```diff
@@ -39,16 +39,35 @@ export default Hello;
...
+If you are familiar with other static site generators like Jekyll and Next, this routing approach will feel familiar to you. Any JavaScript file you create under `/src/pages/` directory will be automatically converted to a website page, following the `/src/pages/` directory hierarchy. For example:
 
-### Using React
+- `/src/pages/index.js` → `<baseUrl>/`
+- `/src/pages/test.js` → `<baseUrl>/test`
+- `/src/pages/foo/test.js` → `<baseUrl>/foo/test`
+- `/src/pages/foo/index.js` → `<baseUrl>/foo`
 
-React is used as the UI library to create pages. You can leverage on the expressibility of React to build rich web content.
+In this era of components, if you need to customize your page design with your own styles, we recommend co-locating your styles with the page component in its own directory. For example, to create a new support page, you could:
```
Comment (wgao19) [inline website/docs/creating-pages.md]: what’s “era of components”?

Comment (endiliey) [reply website/docs/creating-pages.md]: jsx in markdown era

**Comment by wgao19** [comment]
This is neat, thank you!

**Comment by wgao19** [comment]
Referencing #1640

File: website/docs/creating-pages.md (hunk 1 above)
Comment (yangshun) [reply website/docs/creating-pages.md]: You're right. Good catch!

File: website/docs/creating-pages.md (hunk 2 above)
Comment (yangshun) [reply website/docs/creating-pages.md]: Changed it to:

> In this component-based development era, it is encouraged to co-locate your styling, markup and behavior together into components.

**Pull request merged**